package.json

```diff
@@ -23,38 +23,37 @@
     "lint:eslint": "eslint \"**/*.{ts,tsx}\" --max-warnings=0",
     "lint:tsc": "tsc",
     "lint:prettier": "prettier --cache --check .",
-    "analyze": "npx source-map-explorer 'build/static/js/*.js'"
+    "analyze": "pnpm dlx source-map-explorer \"dist/assets/*.js\" --no-border-checks"
   },
   "dependencies": {
-    "@h5web/app": "13.0.0",
-    "@h5web/h5wasm": "13.0.0",
-    "@react-hookz/web": "15.1.0",
+    "@h5web/app": "14.0.0",
+    "@h5web/h5wasm": "14.0.0",
+    "@react-hookz/web": "25.1.0",
     "h5wasm-plugins": "0.0.3",
-    "immer": "9.0.15",
     "normalize.css": "8.0.1",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "react-dropzone": "14.2.1",
-    "react-error-boundary": "4.0.11",
-    "react-hook-form": "7.34.2",
-    "react-icons": "4.4.0",
+    "react": "18.3.1",
+    "react-dom": "18.3.1",
+    "react-dropzone": "14.3.8",
+    "react-error-boundary": "5.0.0",
+    "react-hook-form": "7.54.2",
+    "react-icons": "5.4.0",
     "react-router-dom": "6.3.0",
-    "suspend-react": "0.0.8",
-    "zustand": "4.0.0"
+    "suspend-react": "0.1.3",
+    "zustand": "5.0.3"
   },
   "devDependencies": {
-    "@types/node": "^20.10.5",
-    "@types/react": "^18.2.31",
-    "@types/react-dom": "^18.2.14",
-    "@vitejs/plugin-react": "4.2.1",
+    "@types/node": "^22.13.10",
+    "@types/react": "^18.3.19",
+    "@types/react-dom": "^18.3.5",
+    "@vitejs/plugin-react-swc": "3.8.1",
     "babel-preset-react-app": "10.0.1",
     "eslint": "8.56.0",
     "eslint-config-galex": "4.5.2",
     "npm-run-all": "4.1.5",
-    "prettier": "3.1.1",
+    "prettier": "3.5.3",
     "typescript": "5.0.4",
-    "vite": "5.1.6",
-    "vite-plugin-checker": "0.6.4",
+    "vite": "6.2.2",
```

Comment:
Nothing relevant, I don't think, but it does seem to fix the out-of-memory issue when generating source maps, though it could just be I have more RAM now... https://vite.dev/guide/migration.html#migration-from-v5